src/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function isWindowExists() {
+    return typeof window !== 'undefined';
+}
+
+export const cssClassNames = {
+    loaded: 'ng-lazyloaded',
+    loading: 'ng-lazyloading',
+    failed: 'ng-failed-lazyloaded',
+}
```

Comment:
What do you think of creating a new file, calling constants and place `cssClassNames` inside it?